features/step_definitions/journey_test_steps.rb

```diff
@@ -189,11 +189,11 @@
 
 When /I click the '(.*)' link in the '(.*)' column for the supplier '(.*)'/ do |link_name, column_name, supplier_name|
   @supplierName = supplier_name
+  
+  table_row_first_cell = page.first(:css, "td.summary-item-field-first") {|elem| elem.text == supplier_name}
```

Comment:
Um... it selects only elements where the text matches the supplier name passed in. Looks like that select thing probably?